bin/desi_tsnr_afterburner

```diff
@@ -1183,5 +1174,29 @@ def main():
                 new_tile_table.write(csvtiles, overwrite=True)
                 log.info("wrote {}".format(csvtiles))
 
+            # compute_tile_completeness_table may have found new GOALTIME info
+            # from auxiliary_table_filename, so update exposures if needed
+            for tileid, goaltime in new_tile_table['TILEID', 'GOALTIME']:
+                ii = tsnr2_expid_table['TILEID'] == tileid
```

Comment:
Yes.  Some context:  in the original code, after writing out the `tsnr2_expid_table`, the tiles table logic read it back in from disk but called it `exposure_table` before proceeding (I'm not sure why it didn't just use `tsnr2_expid_table`, but it might because the act of writing it rounded its precision (!) or maybe just historical organically grown code leftovers).  Regardless, I maintained the exposure_table = tsnr2_expid_table alias to minimize number of lines I would change.  But by the time I got here and was adding new code, I decided to update the original name `tsnr2_expid_table` because that is the actual table that is going to get written out, just in case some future change make `exposure_table` a copy or something like that, even if currently the two are aliases for the same thing in memory.